### PR TITLE
Add AIX, ppc64le and s390x as supported architectures

### PR DESCRIPTION
**Pull request by miladfarca**

### CODE REVIEW

**Comment by RReverser** [comment]
@backes Can you take a look please?

**Comment by backes** [comment]
Can we split the list by officially supported and externally maintained architectures? I.e. remove MIPS from the list, and add something like "Additional systems (IBM i, AIX) and processors (MIPS, ppcle64, s390x) are externally maintained, see [ports](https://v8.dev/docs/ports).".

@natorion WDYT?

**Comment by natorion** [comment]
Making it explicit what is supported and an externally maintained/officially unsupported port on the main page would be fine IMO.

**Comment by miladfarca** [comment]
I have separated the supported architectures as suggested, please take a look.

File: src/docs/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This documentation is aimed at C++ developers who want to use V8 in their applic
 
 ## About V8
 
-V8 implements <a href="https://tc39.es/ecma262/">ECMAScript</a> and <a href="https://webassembly.github.io/spec/core/">WebAssembly</a>, and runs on Windows 7 or later, macOS 10.12+, and Linux systems that use x64, IA-32, ARM, or MIPS processors. V8 can run standalone, or can be embedded into any C++ application.
+V8 implements <a href="https://tc39.es/ecma262/">ECMAScript</a> and <a href="https://webassembly.github.io/spec/core/">WebAssembly</a>, and runs on Windows 7 or later, macOS 10.12+, and Linux systems that use x64, IA-32, or ARM processors. Additional systems (IBM i, AIX) and processors (MIPS, ppcle64, s390x) are externally maintained, see <a href="https://v8.dev/docs/ports">ports</a>. V8 can run standalone, or can be embedded into any C++ application.
```
Comment (backes) [inline src/docs/index.md:11]:
```suggestion
V8 implements <a href="https://tc39.es/ecma262/">ECMAScript</a> and <a href="https://webassembly.github.io/spec/core/">WebAssembly</a>, and runs on Windows 7 or later, macOS 10.12+, and Linux systems that use x64, IA-32, or ARM processors. Additional systems (IBM i, AIX) and processors (MIPS, ppcle64, s390x) are externally maintained, see [ports](/docs/ports). V8 can run standalone, or can be embedded into any C++ application.
```

**Review by backes — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**